packages/wonder-blocks-modal/components/one-pane-dialog/one-pane-dialog.js

```diff
@@ -100,12 +115,35 @@ export default class OnePaneDialog extends React.Component<Props> {
         closeButtonVisible: true,
     };
 
+    renderHeader(uniqueId: string): React.Element<typeof ModalHeader> {
+        const {title, subtitle, breadcrumbs} = this.props;
+
+        if (breadcrumbs) {
```

Comment:
`breadcrumbs` is `void | React.Element<typeof Breadcrumbs>` so the type refinement should work but for some reason I had to add a cast below.